package-kubeapps-version.sh

```diff
@@ -201,7 +204,7 @@ update_default_values() {
   local values_file=$1
 
   info "Updating values file to default to carvel support."
-  yq '.packaging.helm.enabled = false , .packaging.carvel.enabled = true' --inplace $values_file
+  yq eval-all --inplace '. as $item ireduce ({}; . *+ $item )' $values_file $default_values_file
```

Comment:
Thought that having a dedicated file for carvel default values might be handy. Happy to remove this improvement if deemed unnecessary 😃

Reply:
There are some nuances when it comes to the way of merging files. E.g. what if there is an array of values and the new data being merged just wants to add instead of replace all items. But for now I think it can work for us.